### ISSUE TITLE
make it easier to read full server error when DatasetScripts.execute() fails

### ISSUE THREAD

**Issue by alextanski** [Crunch-io/scrunch#417]
Currently `execute()` will raise and then sometimes cascade the error if we do not catch it explicitly. We should have an easy way to get the full error message reported by default, because the server error messages are very very useful to quickly finding the fix for any wrong syntax and such:

```
except pycrunch.ClientError as err:
    if err.status_code == 400:  # Script validation
        raise ScriptExecutionError(err)
    raise err  # 404 or something else
```

This bit could always simply raise with whatever the 400 was, instead of using the customized `ScriptExecutionError`.

**Comment by jjdelc** [comment]
The `ScriptExecutionError` has a `.resolutions` attribute that contains the details https://github.com/Crunch-io/scrunch/blob/master/integration/test_scripts.py#L62 - This matches the API response.

**Comment by jjdelc** [comment]
Ok, created PR https://github.com/Crunch-io/scrunch/pull/418 where I address the problem. The bug was that the current code was handling only the case of synchronous validation. The controller will switch to asynchronous validation if the script is roughly bigger than 1MB (2**20) or if the dataset has more than 10K variables.

**Comment by jjdelc** [comment]
Released in v0.10.0
